app/src/main/kotlin/com/wire/android/media/audiomessage/RecordAudioMessagePlayer.kt

```diff
@@ -118,7 +118,7 @@ class RecordAudioMessagePlayer @Inject constructor(
     suspend fun playAudio(
         audioFile: File
     ) {
-        if (currentAudioFile != null) {
+        if (currentAudioFile != null && audioFile.name == currentAudioFile?.name) {
```

Comment:
If current file name is different from what we receive, then we reset the media player and continue with the new received file